simplq/src/components/pages/Admin/index.jsx

```diff
@@ -28,7 +28,7 @@ export default () => {
 
   const [items, setItems] = useState();
 
-  const update = () => {
+  const update = useCallback(() => {
```

Comment:
memoizing these functions and passing them as deps to useEffect will remove the warnings and prevent unwanted network calls

Reply:
What does memoizing the function really mean here? It doesn't return anything so how is it memoized?

Reply:
We need to pass the function ref as deps to useCallback.
If we directly pass a regular function, the reference changes on every re-render, so it would get executed ever time.

On memoization, the function ref changes only when of of the deps specified (queueId here) changes.
So useEffect only runs when queueId (and hence update) changes

Reply:
Ok, I got it

Reply:
Are you sure this will work? Test

Step 1: Create a queue
Step 2: Add three people, and be in the status page of third user. It will show ahead count as 2.
Step 3: Remove the first user from the queue.
Step 4: In the status page of user 3, press refresh. Confirm that ahead count went down to 1.

Reply:
Confirmed that this works as expected.

Do you see any reason otherwise?

Reply:
I was wondering, if the queueId remains the same (which is the case when you click on refresh button), the function will not get called, as we want. can you put a console.log and check?